test.disabled/ejabberd_tests/tests/dynamic_modules.erl

```diff
@@ -10,25 +10,29 @@ save_modules(Domain, Config) ->
 
 ensure_modules(Domain, RequiredModules) ->
     CurrentModules = get_current_modules(Domain),
-    [ensure_module(Domain, Mod, Opts, proplists:get_value(Mod, CurrentModules, stopped))
-        || {Mod, Opts} <- RequiredModules].
+    {ToReplace, ReplaceWith} = to_replace(RequiredModules, CurrentModules, [], []),
+    ok = escalus_ejabberd:rpc(gen_mod_deps, replace_modules,
+                              [Domain, ToReplace, ReplaceWith]).
+
+to_replace([], _CurrentModules, ReplaceAcc, ReplaceWithAcc) ->
+    {lists:usort(ReplaceAcc), ReplaceWithAcc};
```

Comment:
When this list may require removing duplicates? Does `gen_mod_deps:replace_modules` require it to be sorted?

Reply:
I don't remember specifics, but I think the `non-duplicates` condition was vital.

Reply:
I can't see how this Acc may end up with duplicates but OK, won't hurt to ensure this here.